src/main/java/duke/OutOfListException.java

```diff
@@ -0,0 +1,12 @@
+package duke;
+
+public class OutOfListException extends DukeException {
```

Comment:
I like how you specify different kind of exception as child of DukeException (I merely use DukeException itself), it is much neater!